projects/dot-rules/src/lib/services/ConditionGroup.ts

```diff
@@ -157,17 +164,24 @@ export class ConditionGroupService {
 
     private _catchRequestError(operation): Func {
         return (err: any) => {
+            let message = null;
+
             if (err && err.status === HttpCode.NOT_FOUND) {
-                this.loggerService.info('Could not ' + operation + ' Condition: URL not valid.');
+                message = 'Could not ' + operation + ' Condition: URL not valid.';
```

Comment:
labels for this message ?

Reply:
really this a message to show in the console.log, and really this change is not necessary

Reply:
done https://github.com/dotCMS/core-web/pull/1285/commits/fcd832c15fcb74668f088a630f2024edb0f19fae#diff-63d0117db0f2415fab7c86ee3bcb6b8bL167-R172